### ISSUE TITLE
Selected date when editing an appointment

### ISSUE THREAD

**Issue by o11w** [alextselegidis/easyappointments#433]
When a client open an existing appointment form for edition (from the appointment link in the email), the date in the calendar is not correct. (But the json data in appointmentData) is ok.

![capture0](https://user-images.githubusercontent.com/36893815/36740007-91e8639a-1be1-11e8-8183-0ac506430a1a.PNG)
![capture1](https://user-images.githubusercontent.com/36893815/36740026-9a650e56-1be1-11e8-9859-eb19c0be06d2.PNG)
![capture2](https://user-images.githubusercontent.com/36893815/36740031-9d5e6cb0-1be1-11e8-9e59-dfe04e4f033e.PNG)

**Comment by alextselegidis** [comment]
Hi @o11w, can you explain this a bit more? 

> But the json data in appointmentData is ok

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by o11w** [comment]
Hi Alex,
I mean that: in the source of the page, in the javascript, in the declaration of _GlobalVariables_, in the _appointmentData_ row, the values of _start_datetime_ and _end_datetime_ are correct:

```
        "start_datetime": "2018-03-29 15:40:00",
        "end_datetime": "2018-03-29 16:00:00",
```

But in the calendar, within the form, the selected date is the first available of the month.
If the user does not pay attention to this, and does not redefine the date, that date will change to the first of march, and this behaviour will be considered as an error.

Thanks for your help.
Cheers,
Olivier.

**Comment by alextselegidis** [comment]
@o11w I see, thanks for the clarification. 

I will try to reproduce the issue and get back to you. 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by alextselegidis** [comment]
You are totally right, this will be fixed for 1.3.1 

Thanks for posting 👍 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by alextselegidis** [comment]
Pushed in hotfix branch.

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>
